src/components/Navigation.vue

```diff
@@ -33,7 +33,11 @@
               <span class="nav-link-inner-text">Menu</span>
             </a>
             <ul class="dropdown-menu">
-              <li><a class="dropdown-item"><router-link to='/settings'>Settings</router-link></a></li>
+              <li>
+                <a class="dropdown-item"
+                  ><router-link to="/settings">Settings</router-link></a
+                >
```

Comment:
@itisdb please try to check this kind of issues. Maybe you are using some kind of prettier formatter which is creating such messy code.

WRONG:
```
...</a
>
```

CORRECT:
```
...
</a>
```

Reply:
@itisdb unaddressed